index.js

```diff
@@ -177,91 +177,70 @@ module.exports = {
         // prettier-ignore
         if (err) return cb(clarify(err, "Couldn't get own root when excluding members"))
 
-        get(groupId, (err, { writeKey: oldWriteKey } = {}) => {
+        const excludeContent = {
+          type: 'group/exclude',
+          excludes: feedIds,
+          recps: [groupId],
+        }
+        const excludeOpts = {
+          tangles: ['members'],
+          isValid: isExclude,
+        }
```

Comment:
I think we need to think about where we want the exclude published... (which feed).
I have a feeling it should be in the same feed as the additions... but I can't quite justify.
I think it's that we want to be sure we have all the messages for each epoch membership stuff and sprinkling the around might lead to funny edge cases?

what do you think?